instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/__init__.py

```diff
@@ -204,6 +206,7 @@ def _trace_postrun(self, *args, **kwargs):
         # request context tags
         if span.is_recording():
             span.set_attribute(_TASK_TAG_KEY, _TASK_RUN)
+            span.set_attribute(SpanAttributes.MESSAGING_SYSTEM, _QUEUE_NAME)
```

Comment:
I’m not sure if “celery” is the right value here.
According to https://opentelemetry.io/docs/specs/semconv/messaging/messaging-spans/#messaging-attributes
the expected values are Kafka, ActiveMQ, RabbitMQ, etc.
What do you think?

Reply:
Here's a pr for adding celery to semantic conventions https://github.com/open-telemetry/semantic-conventions/pull/1954

Reply:
that requires extracting broker info from the context. The context that we instrument doesn't have that info.

Reply:
Ah, I missed a few more attributes here

https://github.com/open-telemetry/opentelemetry-python-contrib/blob/437648e6680b948926ee78157e01f0ceb73013a8/instrumentation/opentelemetry-instrumentation-celery/src/opentelemetry/instrumentation/celery/utils.py#L95-L116

which also add conversation id and destination name (routing key). 

It does not change a general impression: celery is not a messaging system and does not (potentially cannot) provide messaging-level instrumentation